gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -280,9 +280,10 @@ def __init__(self,
     self._sample_name_encoding = sample_name_encoding
     self._use_1_based_coordinate = use_1_based_coordinate
 
-  def _read_records(self, (file_path, block)):
+  def _read_records(self, xxx_todo_changeme):
     # type: (Tuple[str, Block]) -> Iterable(Variant)
     """Reads records from `file_path` in `block`."""
+    (file_path, block) = xxx_todo_changeme
```

Comment:
Update the name.

Reply:
Oops, Done. now I have to check for this everywhere :'(